src/contracts/proxies/ODSafeManager.sol

```diff
@@ -153,11 +154,15 @@ contract ODSafeManager is IODSafeManager {
 
   /// @inheritdoc IODSafeManager
   function modifySAFECollateralization(
+    address _taxCollector,
```

Comment:
Note that taxCollector can be an arbitrary address inputted by the user still (this was also the case previously in the different functions in `BasicActions.sol`)
So if the tax collector can be multiple addresses, we should either check it is
a valid taxCollector OR if there is only one and we have discoverability,
we should query it here and use that here instead of taking it as a param and remove the `_taxCollector` param here as well as in `BasicActions.sol` to reduce calldata gas costs and another way for users to bypass the tax.